package/crossplane.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: meta.pkg.crossplane.io/v1alpha1
+kind: Provider
+metadata:
+  name: provider-exoscale
+  annotations:
+    meta.crossplane.io/maintainer: VSHN <info@vshn.net>
+    meta.crossplane.io/source: github.com/vshn/provider-exoscale
+    meta.crossplane.io/license: Apache-2
+    meta.crossplane.io/description: |
+      Crossplane provider for managing resources on exoscale.com
+spec:
+  controller:
+    image: ghcr.io/vshn/provider-exoscale/controller:latest
+    permissionRequests: []
+  crossplane:
```

Comment:
This file is dynamically generated, please put it into .gitignore.

Reply:
This file is still present in the PR...

Reply:
I removed it now